ib-create-tag.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import datetime
+import urllib.error
```

Comment:
this does not work for python2. I would suggest to use  
```
from _py2with3compatibility import HTTPError
```
and then replace
```
except urllib.error.URLError:
```
with
```
except HTTPError:
```